core/src/test/java/hudson/scheduler/CronTabDayOfWeekLocaleTest.java

```diff
@@ -50,8 +49,8 @@ public CronTabDayOfWeekLocaleTest(Locale locale) {
      * HUDSON-8656.
      */
     @Test
-    @Url("http://issues.hudson-ci.org/browse/HUDSON-8656")
-    public void hudson8658() throws Exception {
+    @Issue("HUDSON-8656") // This is _not_ JENKINS-8656
+    public void hudson8656() throws Exception {
```

Comment:
Why do you rename the test?

Reply:
Apparent typo in test name. Per the comment this is straight from HUDSON-8656 after the split (and no HUDSON-8658 is archived for us to check).

> <img width="1573" alt="Screenshot" src="https://user-images.githubusercontent.com/1831569/115143756-7cb18e80-a049-11eb-8308-a673a7350ceb.png">